src/omp/exec_dist.f90

```diff
@@ -2,7 +2,6 @@ module m_omp_exec_dist
   use mpi
 
   use m_common, only: dp
-  use m_omp_common, only: SZ
```

Comment:
Not being able to access `SZ` as a parameter will probably make performance worse, because compiler won't be certain whether the `simd` loops can be vectorised or not at compile time.